### PR TITLE
feat:add home page

### PR DESCRIPTION
**Pull request by kavyakunder**
Static Home Page

https://user-images.githubusercontent.com/81958043/161457433-c4a2469f-fb7b-41ba-9c57-3d4707542022.mp4

### CODE REVIEW

File: src/pages/Home/sub-components/reason.jsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import WhatsAppIcon from "@mui/icons-material/WhatsApp";
+import LocalShippingIcon from "@mui/icons-material/LocalShipping";
+import HandshakeIcon from "@mui/icons-material/Handshake";
+import PaymentIcon from "@mui/icons-material/Payment";
```
Comment (dev-enforced) [inline src/pages/Home/sub-components/reason.jsx]: Code suggestion:
Can collect all of these icons in a single icons.js file and then from that icons.js file you can use it anywhere accordingly!!

**Review by dev-enforced — APPROVED** [review]
The code structure looks really good. The naming you have used for CSS classes is easy to understand and up to the point.Gave a minor suggestion on improving the code which is mentioned below. Rest all seems good!!
Hope to see the live preview of Robostore soon!!

File: src/components/Navbar/navbar.css [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+}
+.navbar-logo {
+  height: 3rem;
+  width: 3rem;
+  border-radius: 50%;
+}
+.nav-items {
+  display: flex;
+  align-items: center;
+  font-size: 1.3rem;
```
Comment (stutisk) [inline src/components/Navbar/navbar.css:22]: Can use the font size as variable

**Review by stutisk — APPROVED** [review]
Everything else looks good ,as suggested you can store all your icons used in one file 
Good to merge

**Review by adith-a-danthi — COMMENTED** [review]
The code looks good and well structured. The sections being split into components makes it easy to understand and review. One small suggestion, you can use barrel imports for `components`, and `sub-components`.

**Pull request merged**